feature-utils/poly-import/test/storage.test.js

```diff
@@ -0,0 +1,28 @@
+import { polyProtocolRegex } from "@polypoly-eu/api";
+import { MockPod } from "@polypoly-eu/api/dist/mock-pod";
+import { FeatureFileStorage } from "../src";
+
+describe("File storage ", () => {
+    let fileStorage;
+    beforeAll(() => {
+        const pod = new MockPod();
+        fileStorage = new FeatureFileStorage(pod);
+    });
+    it("can be instantiated", () => {
+        expect(fileStorage.files).toBeNull;
+    });
+});
+
+describe("File storage", () => {
+    let fileStorage;
+    let fileUri;
+    beforeAll(async () => {
+        const pod = new MockPod();
+        fileUri = await pod.polyOut.importArchive("noRealFile.zip");
+        fileStorage = new FeatureFileStorage(pod);
+    });
+    it("ignores non-existent archives", () => {
+        expect(fileUri).toMatch(polyProtocolRegex);
```

Comment:
Why export the regex and test it? I think it'd be better to leave it as an implementation detail and write a dedicated test for handling of invalid URLs. That regex is never used directly in production code, so no point testing it directly IMHO.

Reply:
Actually, testing the fileUri *against* the protocol. The problem with leaving it as an implementation details is that we might end up having incompatible implementation details, and causing issues such as this one https://jira.polypoly.eu/browse/PROD4POD-1133 So it's not so much an implementation details as a single source of truth, because eventually we will be interchanging files between different implementations.